test/ome-files/tiff.cpp

```diff
@@ -153,7 +164,7 @@ namespace
         idx[ome::files::DIM_CHANNEL] = idx[ome::files::DIM_MODULO_Z] =
         idx[ome::files::DIM_MODULO_T] = idx[ome::files::DIM_MODULO_C] = 0;
 
-      const char * const shades[] = {" ", "░", "▒", "▓", "█"};
+      const std::array<const char *,5> shades{{" ", "░", "▒", "▓", "█"}};
```

Comment:
hmmm, we mention in "Contributing to OMERO.insight" (of all places) that "the OMERO Java and Python source files are encoded in UTF-8" -- maybe we need a broader statement among our general contributing docs.